File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Editors Package/ImageList Adv/VB/Docking1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Windows.Forms

Namespace ImageListAdv_Demo
	Partial Public Class Docking1
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
			Dim imgList As ImageList = Me.imageListAdv1.ToImageList()
			imgList.ColorDepth = ColorDepth.Depth32Bit
			Me.dockingManager1.ImageList = imgList

		End Sub
	End Class
End Namespace

File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Editors Package/ImageList Adv/CS/Docking1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

namespace ImageListAdv_Demo
{
    public partial class Docking1 : UserControl
    {
        public Docking1()
        {
            InitializeComponent();
            ImageList imgList = this.imageListAdv1.ToImageList();
            imgList.ColorDepth = ColorDepth.Depth32Bit;
            this.dockingManager1.ImageList = imgList;
        
        }
    }
}
